File: frontend-api/src/main/java/no/nav/melosys/tjenester/gui/dto/MedlemsperiodeDto.java
package no.nav.melosys.tjenester.gui.dto;

import no.nav.melosys.domain.dokument.medlemskap.Periodetype;
import no.nav.melosys.service.kodeverk.KodeDto;

public class MedlemsperiodeDto {

    public PeriodeDto periode;

    public Periodetype medlemtype;

    public KodeDto status;

    public KodeDto grunnlagstype;

    public KodeDto land;

    public KodeDto lovvalg;

    public KodeDto trygdedekning;

    public KodeDto kildedokumenttype;

    public KodeDto kilde;
}

File: frontend-api/src/main/kotlin/no/nav/melosys/tjenester/gui/dto/periode/MedlemsperiodeDto.kt
package no.nav.melosys.tjenester.gui.dto.periode

import no.nav.melosys.service.kodeverk.KodeDto

class MedlemsperiodeDto {

    var periodeID: Long = 0L

    var periode: PeriodeDto? = null

    var periodetype: KodeDto? = null

    var status: KodeDto? = null

    var grunnlagstype: KodeDto? = null

    var land: KodeDto? = null

    var lovvalg: KodeDto? = null

    var trygdedekning: KodeDto? = null

    var kildedokumenttype: KodeDto? = null

    var kilde: KodeDto? = null
}
